src/zatt_converter/zcl_att_conv_zipkin.clas.abap

```diff
@@ -0,0 +1,91 @@
+CLASS zcl_att_conv_zipkin DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    INTERFACES zif_att_trace_converter.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+ENDCLASS.
+
+
+
+CLASS zcl_att_conv_zipkin IMPLEMENTATION.
+
+
+
+  METHOD zif_att_trace_converter~convert.
+
+    DATA(trace_id) = trace->get_trace_id( ).
+
+    DATA(zipkin_json) = |[\{| &&
+                           |"id": "dummy",| &&
+                           |"traceId": "{ trace_id }",| &&
+                           |\}]|.
+
+    converted_trace-trace_id = trace_id.
+    converted_trace-json = zipkin_json.
+
+*  [{
+* "id": "11123456",
```

Comment:
What's with all that json? 🤔

Reply:
That's the reminder that it's not finished. Not good, but maybe better to have some progress.